python/llm/example/GPU/LLM-Finetuning/RewardModeling/reward_modeling_finetuning.py

```diff
@@ -0,0 +1,100 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
```

Comment:
Add link and license to the original file. trl is apache 2.0 license

Reply:
done